packages/node/src/indexer/dictionary/v1/stellarDictionaryV1.ts

```diff
@@ -103,7 +103,11 @@ function eventFilterToQueryEntry(
   const conditions: DictionaryQueryCondition[] = [];
 
   if (Array.isArray(dsOptions)) {
-    const addresses = dsOptions.map((option) => option.address).filter(Boolean);
+    const addresses = dsOptions
+      .map((option) => option.address)
+      .filter(
+        (address): address is string => address !== undefined && address !== '',
+      );
```

Comment:
This will also catch null
```suggestion
      .filter(
        (address): address is string => !!address,
      );
```